WBooks/BookRepository/RentRepository.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  BookRents.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 28/03/2022.
+//
+
+import Foundation
+
+import UIKit
+import Alamofire
+
+public class RentRepository {
+    public func rentBook(onSuccess: @escaping (RentResponse) -> Void,
+                         onError: @escaping (Error) -> Void,
+                         parameters: Rent) {
+        let url = URL(string: "https://private-deb86-wbooksiostraining.apiary-mock.com/users/10/rents")!
```

Comment:
the user id can be sent by parameter so that the function remains dynamic and reusable